spec/javascripts/controllers/inventories/AddInventoryUsersCtrlSpec.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+  'use strict';
+  describe('Controller: AddInventoryUsersCtrl', function() {
+    var $scope, $location, $q, $httpBackend, subject, InventoryParticipant;
+
+    beforeEach(function() {
+      module('PDRClient');
+      inject(function($injector, $controller, $rootScope) {
+        $scope = $rootScope.$new(true);
+        $httpBackend = $injector.get('$httpBackend');
+        InventoryParticipant = $injector.get('InventoryParticipant');
+        $q = $injector.get('$q');
+
+        $scope.inventoryId = 4
+
+        subject = $controller('AddInventoryUsersCtrl', {
+          $scope: $scope
+        });
+
+        $httpBackend.when('GET', '/v1/inventories/4/invitables').respond([]);
+      })
+    });
+
+    it('retrieve initial invitables', function() {
+      $httpBackend.expectGET('/v1/inventories/4/invitables').respond([]);
+      $httpBackend.flush();
+    });
+
+    describe('#addUser', function() {
+      beforeEach(function() {
+        spyOn(subject, 'reloadInvitables').and.callThrough();
+        spyOn(InventoryParticipant, 'create').and.callFake(function(query, params) {
```

Comment:
In this scenario, if you only want to test that the methods are invoked, you could still use `$httpBackend` to accomplish the request going through.

It'd be something similar to...

```$httpBackend.when('GET', '/v1/inventories/4/invitables').respond(200);```

...since you don't need the return value.